packages/@aws-cdk/aws-fsx/lib/lustre-file-system.ts

```diff
@@ -21,7 +21,12 @@ export enum LustreDeploymentType {
   /**
    * Long term storage. Data is replicated and file servers are replaced if they fail.
    */
-  PERSISTENT_1 = 'PERSISTENT_1'
+  PERSISTENT_1 = 'PERSISTENT_1',
+  /**
+   * Newer type of long term storage with higher throughput tiers.
+   * Data is replicated and file servers are replaced if they fail.
+   */
+  PERSISTENT_2 = 'PERSISTENT_2'
 }
 
 /**
```

Comment:
I see there are some unit test that use this enum [here](https://github.com/aws/aws-cdk/blob/0e08eebd2f13ab0da6cac7b91288845cad530192/packages/%40aws-cdk/aws-fsx/test/lustre-file-system.test.ts#L132-L155).

Let's add one that uses `LustreDeploymentType. PERSISTENT_2`.